src/components/DisplayTodo.js

```diff
@@ -1,16 +1,30 @@
 // eslint-disable-next-line
 import React from 'react';
+// eslint-disable-next-line
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+// eslint-disable-next-line
+import { icon } from '@fortawesome/fontawesome-svg-core/import.macro'; // <-- import styles to be used
 
-const DisplayTodo = () => {
+const DisplayTodo = (props) => {
+  const { onHandleCheckBoxChange, onDeleteHandler } = props;
   const todos = JSON.parse(localStorage.getItem('todos'));
 
-  const todoList = todos.map((todo) => (
-        <div className='content-container' key={todo.id}>
-            <input type='checkbox' />
-            <p className='content-paragraph'>{todo.content}</p>
+  const onhandleDelete = (id) => {
+    onDeleteHandler(id);
+  };
 
+  let todoList = null;
+
+  if (todos !== null) {
+    todoList = todos.map((todo) => (
```

Comment:
You can use the concept of destructuring to avoid writing extra code afterward: 
`todoList = todos.map(({id, content}) => (`